docs/docs/channels/sms/_category_.json

```
@@ -0,0 +1,4 @@
+{
+  "label": "SMS",
+  "position": 3
```

```suggestion
  "position": 5
```
That's the current position of the SMS category in the documents. 3 belongs to Push Notifications.